### PR TITLE
Add spanish translation for-the-future-of-bitcoin-wallets

### PR DESCRIPTION
**Pull request by BlueeeMoon**

### CODE REVIEW

File: baltic-honeybadger/2018/the-future-of-bitcoin-wallets.es.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: El futuro de los monederos de Bitcoin
+transcript_by: Bryan Bishop
+translation_by: Blue Moon
+categories: ['conferencia']
+tags: ['hardware wallet', 'wallet']
```
Comment (adamjonas) [inline baltic-honeybadger/2018/the-future-of-bitcoin-wallets.es.md:6]: I think we need to consolidate the tags: https://btctranscripts.com/es/tags/

There is both cartera-hardware and hardware wallet

**Comment by adamjonas** [comment]
You seem to be doing this whole conference. Can you add an `_index.es.md` file in this or your next PR. None of these are indexed at https://btctranscripts.com/es/baltic-honeybadger/

File: baltic-honeybadger/2018/the-future-of-bitcoin-wallets.es.md (hunk 1 above)
Comment (BlueeeMoon) [reply baltic-honeybadger/2018/the-future-of-bitcoin-wallets.es.md:6]: > I think we need to consolidate the tags: https://btctranscripts.com/es/tags/
> 
> There is both cartera-hardware and hardware wallet

Okay, it seems to me that you could stay alone: cartera, do you agree?

**Comment by BlueeeMoon** [comment]
> Parece que estás haciendo toda esta conferencia. ¿Puedes agregar un `_index.es.md`archivo en este o en tu próximo PR? Ninguno de estos está indexado en https://btctranscripts.com/es/baltic-honeybadger/

Sure, I'll add it in the next one, thanks.

**Comment by BlueeeMoon** [comment]
I have updated the name of the labels. Thanks!!

**Pull request merged**